### ISSUE TITLE
Get rid of project-le-ssl.conf template

### ISSUE THREAD

**Issue by boatrite** [tenforwardconsulting/subspace#7]
Instead, copy the non-ssl one and use ansible's lineinfile to add the ssl lines as needed.

**Comment by boatrite** [comment]
Somewhat complicated. I was thinking something like [this SO post suggests](http://stackoverflow.com/questions/24003880/ansible-set-variable-to-file-content/25746069#25746069): Using [fetch](http://docs.ansible.com/ansible/fetch_module.html) and [lookup](http://docs.ansible.com/ansible/playbooks_lookups.html#intro-to-lookups-getting-file-contents).

The thing is that we need to remove the `<VirtualHost *:80>` and `</VirtualHost>` lines and instead surround the file contents with
```
<IfModule mod_ssl.c>
    <VirtualHost *:443>
```
and
```
    </VirtualHost>
</IfModule>
```

**Comment by samsonasu** [comment]
yes, I think the solution here is to have apache have a specific template that it uses if ssl is enabled.  It should pull the cert from the same place, `/etc/ssl/server_name/`.  Then the LetsEncrypt role should simply put the correct certs in the correct place and it will work. 

We can even have the apache role generate a self-signed certificate by default so that it's configuration is valid, and then have either letsencrypt or a custom ssl replace it and it will all Just Work ™️ 

This will allow us to use wildcard certs (which are not currently supported by letsencrypt) and it will also help begin to solve the multi-home site issues (e.g we dont want to have www.myblog.com and www.completelydifferentsite.com share the same cert on a CMS).
